docs/workflow.rst

```diff
@@ -238,12 +233,13 @@ Let's get in more detais:
 
```

Comment:
Always keep in mind that you need to rebase often the work of that upstream branch to your working branch. => keep the sentence simple.

always rebase your workspace branch with the upstream branch as often as possible.